src/style.css

```diff
@@ -0,0 +1,95 @@
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir-Book;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 900;
+}
+
+* {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+.search-box {
+  margin-top: 20px;
+  margin-bottom: 20px;
+  margin-left: 8px;
+  margin-right: 8px;
+}
+
+.search-input {
+  font-family: Avenir-Heavy, sans-serif;
+  width: 100%;
+  height: 70px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  background-image: url("./images/Search.svg");
+  background-position: 25px 24px ;
+  background-size: auto;
+  font-size: 16px;
+  background-repeat: no-repeat;
+  text-indent: 61px;
+}
+
+.search-input::placeholder {
+  font-family: Avenir-Book, sans-serif;
+  font-size: 16px;
+  line-height: 70px;
+  color: #3d4e61;
+  /* transform: translateY(1px); */
+}
+
+.search-input--small::placeholder {
+  font-family: Avenir-Book, sans-serif;
```

Comment:
u can move this property to `html` selector, and use `inherit` everywhere else where needed

Reply:
moved property to html